### PR TITLE
Update README for newer TRTIS releases

### PR DESCRIPTION
**Pull request by deadeyegoodwin**
TensorRT Inference Server requires new handling starting with the 18.11 release. The current README only works with older versions of the server and so needs this update to be relevant.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2279)

### CODE REVIEW

File: kubeflow/nvidia-inference-server/README.md [hunk 1]
```diff
@@ -23,23 +23,20 @@ CPUs at scale.
...
 `io.ksonnet.pkg.nvidia-inference-server` prototype to generate
 Kubernetes YAML and deploy to that cluster.
 
-For more information on the NVIDIA Inference Server see the [NVIDIA
-Inference Server User
-Guide](https://docs.nvidia.com/deeplearning/sdk/inference-user-guide/index.html)
-and the [NVIDIA Inference Server
-Clients](https://github.com/NVIDIA/dl-inference-server) open-source
-repository.
+TRTIS is open-source software that also releases monthly as prebuild,
```
Comment (pdmack) [inline kubeflow/nvidia-inference-server/README.md]:
```suggestion
TRTIS is open-source software that also releases monthly as prebuilt,
```

**Comment by pdmack** [comment]
If you can't address the CLA issue as noted, we'll need a project owner like @jlewi to work some magic.

**Comment by pdmack** [comment]
Hmmm, yeah you're right. Unfortunate side effect.

**Comment by pdmack** [comment]
Can you try a "no-op" commit on the PR to turn it back to you? Maybe some whitespace.

**Comment by deadeyegoodwin** [comment]
@pdmack OK, I fixed the CLA by removing your commit and just folding the change into the original commit. Should be unblocked now for approval and commit.

**Pull request merged**